decidim-accountability/app/views/decidim/accountability/admin/timeline_entries/edit.html.erb

```diff
@@ -1,3 +1,5 @@
+<% add_decidim_page_title(t(".title")) %>
+<% add_decidim_page_title(translated_attribute(result.title)) %>
```

Comment:
Lets respect the current `item_show__header-title`, as that's whats shown in the HTML: 
```suggestion
```

![Screenshot of this page in a browser](https://github.com/decidim/decidim/assets/717367/9ee00b78-06c3-4fb3-a3e9-9cd1ac9ac0c0)